azure/src/tree/AzureAccountTreeItemBase.ts

```diff
@@ -92,9 +94,20 @@ export abstract class AzureAccountTreeItemBase extends AzExtParentTreeItem imple
                 iconPath: new ThemeIcon('loading~spin')
             })];
         } else if (azureAccount.status === 'LoggedOut') {
+            const studentAccountTreeItem =new GenericTreeItem(this, { 
+                label: createStudentAccountLabel,
+                 commandId: 'azureResourceGroups.openUrl', 
+                 contextValue, 
+                 id: createStudentAccountCommandId, 
+                 iconPath: new ThemeIcon('mortar-board'), 
+                 includeInTreeItemPicker: true});
```

Comment:
alignment of these lines

<img width="1201" alt="image" src="https://user-images.githubusercontent.com/12476526/168398304-16eaee72-a808-4605-8631-15bef68f4da2.png">

Reply:
So bizarre...